### ISSUE TITLE
TUF-AX5400 DDNS causes disconnection

### ISSUE THREAD

**Issue by bobyang3** [gnuton/asuswrt-merlin.ng#317]
**Is your feature request related to a problem? Please describe.**
I am not sure if this is a bug or not or only helps are needed so I don't log this as we bug. Currently, *I think*. I got disconnected when the router failed on DDNS update. I don't think it is hardware issue because CPU and memory are good.

![image](https://user-images.githubusercontent.com/2717137/211089169-c1456bbb-f25d-4281-a7ff-a633a6eba079.png)

![image](https://user-images.githubusercontent.com/2717137/211089204-a2ae9f86-5e26-49b6-b993-5533f581df82.png)

**Comment by gnuton** [comment]
I do not think DDNS is related to any disconnection.
You got some requet timeout but since you get only one and not multiple and witout resolve or routing errors I do not thnk you got disconnected either

**Comment by bobyang3** [comment]
Thank you. Agreed, this is very odd to see failed DDNS will cause this
issue. I always RDP to my work laptop (both computers are at home) and
that's how I found I was not getting ping response from the router and
google DNS. I also ping -t at my work laptop, and I saw it failed to the
router every 5 mins as well.

It was like that for about 30 mins. then I disabled DDNS, and disabled
self-cert. Then, disconnection issues disappeared. I re-enable it again
with the new updated password, so DDNS won't fail. everything is back to
normal. I don't see a non-response ping from the router anymore. Just feel
weird and interesting
